internal/proxy/oauth2.go

```
@@ -77,14 +77,13 @@ func (h *HttpProxy) RegisterOrUpdate(dst *OAUTH2Proxy) error {
 			v.Paths = dst.Paths
 
-			return nil
+			return
 		}
 	}
 
 	h.log.Info("register http backend", "host", dst.Host, "service", dst.Service, "port", dst.Port)
 	h.dst = append(h.dst, dst)
-
-	return nil
+	return
```

golangci-lint (2.12.2)

[error] S1023: redundant return statement

(staticcheck)